src/sass/partials/_products.scss

```diff
@@ -0,0 +1,215 @@
+        gap: 130px;
+        max-width: 280px;
+        margin: 144px auto 98px auto;
+    }
+
+    &__card {
+        width: 100%;
+        height: 485px;
+        background-color: $IBgColor;
+        border-radius: 24px;
+        position: relative;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+
+
+        &--yellow {
+            background-color: $IIBgColor;
+        }
+    
+        &--green {
+            background-color: $IIIBgColor;
+        }
+    }
+
+    &__card::before {
+        content: "";
+        display: block;
+        background-image: url("/src/images/home-pic1.png");
+        width: 249px;
+        height: 252px;
+        position: absolute;
+        background-size: cover;
+        top: 0;
+        left: 50%;
+        transform: translate(-50%, -45%);
+    }
+
+    &__card--yellow::before {
+        background-image: url("/src/images/home-pic2.png");
+    }
+
+    &__card--green::before {
+        background-image: url("/src/images/home-pic3.png");
+    }
+
+    &__name {
+        color: $IColor;
+        font-size: 20px;
+        text-transform: uppercase;
+        margin-top: 161px;
+
+    }
+
+    &__name::after {
+        content: "";
+        display: block;
+        width: 30px;
+        margin: auto;
+        margin-top: 30px;
+        border-bottom: 6px dotted $IColor;
+    
+    }
+
+    &__description {
+        color: $IColor;
+        font-size: 14px;
+        font-family: $fontPrimary;
+        line-height: 176%;
+        text-align: center;
+        margin: 30px 40px 30px 40px;
+    }
+
+}
+
+@media screen and (min-width: 768px) {
+
+    .products {
+    
+        &__header {
+            font-size: 48px;
+        }
+    
+        &__label {
+            font-size: 26px;
+            margin-top: 122px;
+
+        }
+    
+        &__container {
+            flex-wrap: nowrap;
+            gap: 20px;
+            justify-content: center;
+            margin: 150px auto 112px auto;
+            max-width: none;
+        }
+    
+        &__card {
+            width: 33%;
+            max-width: 220px;
+        }
+    
+        &__card::before {
+            background-image: url("/src/images/home-pic1.png");
+            width: 249px;
+            height: 252px;
+        }
+    
+        &__card--yellow::before {
+            background-image: url("/src/images/home-pic2.png");
+        }
+    
+        &__card--green::before {
+            background-image: url("/src/images/home-pic3.png");
+        }
+    
+        &__name {
+            color: $IColor;
+            font-size: 20px;
+            text-transform: uppercase;
+            margin-top: 161px;
+    
+        }
+    
+        &__description {
+            margin: 30px 11px 40px 11px;
+
+            .products_description-hidden {
+                display: none;
+            }
+        
+        }
+
+    }
+
+}
+
+@media screen and (min-width: 1280px) {
```

Comment:
In figma there is 1200px screen breakpoint but in task files there is 1280. Just we need to check it and use one value in all branches.

Reply:
We are using 1280px